test/functional/blockchain.py

```diff
@@ -40,8 +40,15 @@ def setup_network(self, split=False):
     def run_test(self):
         self._test_gettxoutsetinfo()
         self._test_getblockheader()
+        self._test_getdifficulty()
         self.nodes[0].verifychain(4, 0)
 
+    def _test_getdifficulty(self):
+        res = self.nodes[0].getdifficulty()
```

Comment:
Mention the `getdifficulty` RPC test in the top level comment or remove the list of tested methods there.